tests/10-create.js

```diff
@@ -52,14 +51,14 @@ describe('ecdsa-2019 (create)', function() {
             verificationMethodDocuments.push(verificationMethodDocument);
           }
         });
-        it('The field "cryptosuite" MUST be "ecdsa-2019".', function() {
+        it('The field "cryptosuite" MUST be "ecdsa-rdfc-2019" or ' +
+          '"ecdsa-jcs-2019".', function() {
           this.test.cell = {columnId: name, rowId: this.test.title};
+          const cryptosuite = ['ecdsa-rdfc-2019', 'ecdsa-jcs-2019'];
```

Comment:
Missing a cryptosuite here:

```suggestion
          const cryptosuite = ['ecdsa-rdfc-2019', 'ecdsa-jcs-2019', 'ecdsa-sd-2023'];
```

Reply:
Updated to include `ecdsa-sd-2023` [f05fa7c](https://github.com/w3c/vc-di-ecdsa-test-suite/pull/13/commits/f05fa7c94012ad2e13a8531c6d23e6dfa9f7e083)